src/components/ApplicationPrompt.astro

```diff
@@ -1,13 +1,15 @@
 ---
 import { DateFormatters } from '~utils';
 export interface Props {
-	applicationWindow: [Date, Date];
+	cohortApplicationWindow: [Date, Date];
 	cohortKickoffDate: Date;
+	cohortRegions: string[];
```

Comment:
❓ @segdeha, should we make an explicit type for the strings that are allowed to go into this array, so that we're forced to be consistent each time we name a region? e.g.:

```ts
type CohortRegions = 'North America' | 'Europe/Africa' | 'Asia/Pacific' // etc
```

Reply:
We could, yes. Good idea!

Current values would be `"North America" | "Europe/Africa" | "Latin America"`